lib/chef/knife/bootstrap.rb

```diff
@@ -368,7 +369,7 @@ class Bootstrap < Knife
         ssl_peer_fingerprint:
           [:winrm_ssl_peer_fingerprint, "--ssl-peer-fingerprint FINGERPRINT"],
         winrm_user:
-          [:connection_user, "--winrm-user USER"],
+          [:connection_user, "--winrm-user USER", nil, "-x USERNAME"],
```

Comment:
seems like both options should use either USER or USERNAME here.

Reply:
As per the previous implementation, I used [-x USERNAME](https://github.com/chef/chef/blob/chef-14/lib/chef/knife/bootstrap.rb#L48). If we choose to use USERNAME here, should we also update the same for ssh_user option ?

Reply:
Let's make it consistently USERNAME across connection-user, winrm-user, ssh-user